apps/yapms/src/lib/components/modals/ModalBase.svelte

```diff
@@ -14,18 +13,19 @@
 		} else if ($store !== undefined) {
 			$store.open = false;
 		}
-		console.log(open);
 	}
 
 	let content: HTMLDivElement | undefined;
 	let offsetHeight: number;
+	let dialog: HTMLDialogElement;
 
 	$: isOverflow = offsetHeight < (content?.scrollHeight ?? 0);
-</script>
 
-<input type="checkbox" class="modal-toggle" checked={$store?.open ?? false} />
+	$: if ($store?.open) dialog?.showModal();
+	$: if (!$store?.open) dialog?.close();
```

Comment:
The store types undefined type can be removed, as it's required to be in all modals to function.